README.md

```diff
@@ -1,31 +1,302 @@
-# stolon-pgbouncer
+# stolon-pgbouncer [![CircleCI](https://circleci.com/gh/gocardless/stolon-pgbouncer/tree/master.svg?style=svg)](https://circleci.com/gh/gocardless/stolon-pgbouncer/tree/master)
+'stolon-pgbouncer' extends a highly-available PostgreSQL setup (managed by
+[stolon](https://github.com/sorintlab/stolon)) enabling PgBouncer proxying  and
+zero-downtime failover of the PostgreSQL primary.
 
-## Tests
+See [Playground](#playground) for how to start a Dockerised three node stolon PostgreSQL
+cluster utilising `stolon-pgbouncer`.
 
-Run unit tests:
-    make test
+- stolon-pgbouncer
+    - [Overview](#overview)
+    - [Architecture](#architecture)
+    - [Commands](#commands)
+- [Playground](#playground)
+    - [Architecture](#playground-architecture)
+    - [Installation](#playground-installation)
+    - [Commands](#playground-commands)
 
-Run acceptance tests:
-    make test-acceptance
+- [Development](#development)
+    - [Images](#images)
+    - [Releasing](#releasing)
 
-This will spin up a cluster using docker-compose and run acceptance tests
-against it.  Right now there's a race between the cluster becoming stable and
-the test starting, so it may be a bit flaky.
+### Overview
+GoCardless runs a highly available PostgreSQL cluster using
```

Comment:
This is the overview, so it should be terse and to the point. We don't need to justify any of our design decisions here, or go into detail.

I'm going to sketch out removing any justifications.